packages/compass-crud/src/components/bulk-update-dialog.tsx

```diff
@@ -77,13 +77,22 @@ const multilineContainerStyles = css({
 
 // We use custom color here so need to disable default one that we use
 // everywhere else
-const codeEditorStyles = css({
-  background: 'transparent !important',
+const codeEditorDarkStyles = css({
```

Comment:
ok for a follow up, is this still nothing we could factor in the editor component?

Reply:
Yes that was the decision when we last discussed https://jira.mongodb.org/browse/COMPASS-7352. We seem to always do this light grey slimline card editor that looks a bit like the leafygreen Code component. We seem to now be standardising on the pattern of leaving space at the bottom of it for a syntax error or server/driver error banner as well. All that should probably be part of the standard multiline editor.

Reply:
(Or at least a standardised variant if we still need the white one somewhere.)

Reply:
I asked before and I'll ask again: the main reason we are doing these overrides is because editor component uses background colors different in every design that we get (and this time it's only dark mode, so the light mode override with `important` doesn't actually change the value from the original one). Why can't we align that? 

Like yes, extending editor to support banner errors / warnings makes sense, but here specifically the only reason we are doing this color override that broke editor styles in the first place is because in dark mode color is slightly different in this new design. It's barely noticeable, so if we want to always use this color, let's just make it a default one in the editor

Reply:
I now can't recreate the colours looking wrong so I just removed them.

Reply:
I think it's dark mode only where it's mismatched, this is how it looks after I pulled latest changed:

![image](https://github.com/mongodb-js/compass/assets/5036933/a9931a93-2bdf-4b09-b246-50117efe215c)

I feel like the problem here is that we're using leafygreen code editor color exports that can't be used easily elsewhere because they are just numbered, so very hard to understand what's supposed to be used. I feel like we can just re-export code editor pallette from the package and apply those to the container instead of overriding the editor while we are waiting for [COMPASS-7352](https://jira.mongodb.org/browse/COMPASS-7352) to be implemented